pyzeebe/__init__.py

```diff
@@ -1,13 +1,15 @@
-__version__ = "2.3.1"
+__version__ = "3.0.0"
```

Comment:
I think it could be an idea to release this as a "beta" release, so we can add additional breaking changes after having tested it in production.